variables.tf

```diff
@@ -164,6 +164,18 @@ variable "authorized_networks_read_replica" {
   default = []
 }
 
+variable "region_master_instance" {
+  description = "The region to launch the master instance in. Defaults to the Google provider's region if nothing is specified here. See https://cloud.google.com/compute/docs/regions-zones"
+  type        = string
+  default     = ""
+}
+
+variable "region_read_replica" {
```

Comment:
Do we need to give this as an option? Normally read replicas should be in colocated in zones in the same region as master is. Region wise separation most of the time happen when it's a DR setup. Here it should always provision replicas in the region that's selected for master.

Reply:
We have a use case with one of our projects. This is why this PR was created. Will message privately